src/js/common.js

```diff
@@ -88,6 +88,42 @@ cm.getUserName = function () {
 }
 
 /*** new ***/
+$(function () {
+    setIdleTime();
+})
+// This method checks idle time
+// Check 5 minutes before session expires
+setIdleTime = function () {
+    cm.refreshToken();
+    setInterval(cm.checkIdleTime, 3300000);
```

Comment:
Sprint meeting decided not to check activities and always refresh token 30 minutes.